### ISSUE TITLE
Create a Paragraph component

### ISSUE THREAD

**Issue by dryhten** [elifesciences/elife-xpub#230]
To account for body styles in the [styleguide](https://projects.invisionapp.com/share/NZKXOQUV4P3#/screens/302926983) (see Typography) we would probably need a new Paragraph component.

I would place this one in `atoms`.

**Comment by jsms90** [comment]
@hdrury1 Which part of the existing designs does this refer to? I'm guessing this isn't in the wizard? Is it for the Manuscript preview?

**Comment by hdrury1** [comment]
Correct, this is not a component in the wizard currently. We can move to Backlog.

**Comment by hdrury1** [comment]
I'm sure this has been done- can someone confirm?

**Comment by diversemix** [comment]
@jsms90  was this done by #395 ?
